oga/frontend/src/containers/Answer/NewAnswer.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import './NewAnswer.css';
+
+import { connect } from 'react-redux';
+import * as actionCreators from '../../store/actions/';
+
+import { Redirect } from 'react-router-dom';
+import { push } from 'connected-react-router';
+import Map from '../Map/GoogleMap';
+import PushNotification from '../../components/PushNotification/PushNotification'
+import NewQuestion from '../QuestionList/NewQuestion/NewQuestion';
+
+class NewAnswer extends Component {
+  state = {
+    answer_content: '....',
+  }
+
+  componentDidMount() {
+    this.props.onGetQuestion(this.props.match.params.id);
+  }
+
+  postQuestionHandler = () => {
+    if (this.state.answer_content !== '' && this.props.selectedQuestion.content)
+    { // for testing purposes, we set type to 0, and pass content as well
+      // actually, we only have to store type in questions, 
+      // as content is fixed based on type
+      this.props.createAnswer(this.props.selectedQuestion.content, this.state.answer_content);
+    }
+  }
+
+  clickBackHandler = () => {
+    this.props.history.goBack();
+  }
+
+  render() {
+    let place_name = "...";
+    if (this.props.target_location)
+    {
+      place_name = this.props.target_location.name;
+    }
+    let show = null;
+    var qs_type = this.props.selectedQuestion.content;
+    if (qs_type == "LINE") {
```

Comment:
I think there must be a better way to handle this.
For example, a const table that returns possible answers lists from qs_type?

Reply:
That is good idea. I will make it.